.circleci/config.yml

```diff
@@ -236,6 +236,7 @@ jobs:
           working_directory: packages/contracts-bedrock
       - run:
           name: test and generate coverage
+          no_output_timeout: 20m
```

Comment:
Intended to fix [this failure](https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/9873/workflows/1e515c93-6384-49e1-842c-7eb958f2e0b8/jobs/230614?invite=true#step-107-15).